src/components/LocaleSelector.tsx

```diff
@@ -1,21 +1,55 @@
-import { DropdownItem } from '@terraware/web-components';
+import { DropdownItem, Icon } from '@terraware/web-components';
 import { supportedLocales } from '../strings/locales';
 import { LocalizationContext } from '../providers/contexts';
-import Dropdown from './common/Dropdown';
 import { UserService } from 'src/services';
 import { useUser } from '../providers';
+import { makeStyles } from '@mui/styles';
+import { IconButton, Theme } from '@mui/material';
+import { useState } from 'react';
+import PopoverMenu from './common/PopoverMenu';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  iconContainer: {
+    height: '48px',
+    borderRadius: '16px',
+    padding: theme.spacing(1.5, 2),
+  },
+  icon: {
+    width: '32px',
+    height: '32px',
+  },
+  chevronDown: {
+    marginLeft: '8px',
+    fill: theme.palette.TwClrIcn,
+  },
+  selected: {
+    fontSize: '16px',
+    paddingLeft: '8px',
+    color: theme.palette.TwClrTxt,
+  },
+}));
 
 export default function LocaleSelector(): JSX.Element {
   const { user, reloadUser } = useUser();
   const localeItems: DropdownItem[] = supportedLocales.map((supportedLocale) => ({
     value: supportedLocale.id,
     label: supportedLocale.name,
```

Comment:
unrelated to your PR but this should probably come from `strings`

Reply:
actually no, i am wrong here.. i guess each language should show up in it's respective recognizable script. Nothing to do here, but maybe gibberish should be in gibberish. When we show spanish menu item maybe it will show `española` if that's the correct spelling :)
ignore my comments in this thread

Reply:
It would be `español` haha